src/nvim/ex_getln.c

```diff
@@ -2990,7 +2990,7 @@ void ui_ext_cmdline_block_append(int indent, const char *line)
   memcpy(buf + indent, line, strlen(line));  // -V575
 
   Array item = ARRAY_DICT_INIT;
-  ADD(item, DICTIONARY_OBJ((Dictionary)ARRAY_DICT_INIT));
+  ADD(item, INTEGER_OBJ(0));
```

Comment:
I guess no client is using the feature currently, but in theory should this be listed on https://github.com/neovim/neovim/wiki/Following-HEAD ?

Reply:
This is opt-in, not a breaking change (`api/ui.c` handles the back-compat).